src/main/java/com/google/api/codegen/discovery/config/SampleConfig.java

```diff
@@ -87,6 +87,10 @@
   @JsonProperty("packagePrefix")
   public abstract String packagePrefix();
 
+  /** Returns a sample application name. */
+  @JsonProperty("appName")
+  public abstract String appName();
```

Comment:
Yes, more consistent that way, I was just trying to avoid copying more code. We can do this for now, but I'd like to de-dupe the SampleMethodToViewTransformers once they're done.